docs/source/configuration/events.mdx

```diff
@@ -0,0 +1,195 @@
+---
+title: Events
+subtitle: Log or trace events in the router's request lifecycle
+description:
+---
+
+Events are used to signal when something of note happens in the [Apollo Router's request lifecycle (pipeline)](../customizations/overview/#the-request-lifecycle). Events are output to both logs and traces.
+
+The router's pipeline has three major services:
```

Comment:
Does this stuff work on non-major (minor?) services? If so, may be worth making that clear by something like:

Typically events are configured on the most widely used router services: `Router`, `Supergraph` and `Subgraph`. (Events may be configured on all services supported by the router)

also: Maybe add links to the where the widely used services are documented elsewherei n the router docs.